_includes/docs/0.4.x/cloud-foundry.html

```diff
@@ -40,6 +40,11 @@
         <li><pre>$ grunt build</pre></li>
       </ul>
     </li>
+    <li>Run the Gulp Build task to build the optimized JavaScript and CSS files (If you chose Gulp instead of Grunt)
```

Comment:
I'd prefer to leave this out of the deployment sections until we switch over to gulp completely. 

@lirantal Thoughts?